workflows/tomte.nf

```diff
@@ -220,22 +207,30 @@ workflow TOMTE {
     )
     ch_versions = ch_versions.mix(IGV_TRACKS.out.versions)
 
-    //
-    // Collate and save software versions
-    //
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    COLLECT SOFTWARE VERSIONS & MultiQC
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
     softwareVersionsToYAML(ch_versions)
-        .collectFile(storeDir: "${params.outdir}/pipeline_info", name: 'tomte_software_mqc_versions.yml', sort: true, newLine: true)
-        .set { ch_collated_versions }
+        .collectFile(
+            storeDir: "${params.outdir}/pipeline_info",
+            name:  ''  + 'pipeline_software_' +  'mqc_'  + 'versions.yml',
```

Comment:
```suggestion
            name:  ''  + 'pipeline_software_' +  'mqc_'  + 'versions.yml',
```

You can fill in the first `''` if you want, nextflow pipelines have `nf-core`.

Reply:
It does look neck with tomte there :)